plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/AcunetixReader.java

```diff
@@ -187,6 +187,10 @@ private TestCaseResult parseAcunetixVulnerability(Node vuln) throws Exception {
     //    <IsFalsePositive><![CDATA[False]]></IsFalsePositive>
     //    <Severity><![CDATA[medium]]></Severity>
     //    <CWE id="352"><![CDATA[CWE-352]]></CWE>
+    //  This is the newer <ReportItem> format
+    //    <CWEList>
```

Comment:
Does this mean that there can be several CWEs in one result? Or is this just new wrapping?

Reply:
It seems that this is just the new wrapping as the result XML.
<img width="722" alt="image" src="https://user-images.githubusercontent.com/68040445/227820978-9be47ae2-2072-49ca-8af4-3fa62640b686.png">